app/repositories/beatmap_mirror_requests.py

```diff
@@ -49,9 +49,9 @@ async def get_mirror_weight(mirror_name: str) -> int:
     if failure_rate is None:
         return MIRROR_INITIAL_WEIGHT
 
-    # https://www.desmos.com/calculator/0am8xnwxyo
-    latency_weight = 1000 / math.log(p90_success_ms_latency)
-    failure_weight = math.exp(-10 * failure_rate)
+    # https://www.desmos.com/calculator/wxpsjhdby9
+    latency_weight = 1000 * math.exp(-1 / 1000 * p90_success_ms_latency)
+    failure_weight = math.exp(-30 * failure_rate)
```

Comment:
Failure now has a ~3x higher impact on the overall weight. I found in practice it was not high enough, and in my testing I was assuming a ~10% failure rate to be average, which does NOT reflect reality.

Now in the updated formula, you can easily see how quickly an increased failure rate (x axis) will lead to a decreased weighting (y axis): 

![image](https://i.cmyui.xyz/Wh5byfYf8Ko.png)